### DISCUSSION TITLE
AOE overlays for colorblindness

### DISCUSSION THREAD

**Discussion by y0urd0g** [blish-hud/Blish-HUD#898] [Module Ideas]
I have partial colorblindness and i find myself taking a lot of damage from AOEs that are just so insanely difficult to see for me. It would be super cool if we could get a module that places an overlay over positive and negative AOEs to make them easier to see for us colorblind folk. The ability to customize colors and opacity would be an awesome bonus as well.

**Comment by eksime** [comment]
I can't speak for module developers, but I think this would be quite hard to implement as a Blish module in a way that works nicely. It may be difficult or impossible to determine when an AoE has been cast, and know the location of an AoE field in-game. Additionally, Blish can't outright replace ingame graphics, it can only overlay ingame graphics, so they would need to line up fairly well and I'm pretty sure that information isn't available to module developers.

It may be possible to use a program like reshade or uMod to replace the texture or shaders for the AoE indicators, however doing so is against the terms of service and risks a ban- reshade and similar are usually fine, but messing with aoe/targeting indicators, especially in pvp modes, could be considered gaining a competitive advantage.

It may be worth sending an email to gw2 support or starting a thread on the [forums](https://en-forum.guildwars2.com/) requesting this feature. 

You're likely already aware but Windows has [color filters](https://support.microsoft.com/en-us/windows/use-color-filters-in-windows-43893e44-b8b3-2e27-1a29-b0c15ef0e5ce) that can help a bit with some types of colorblindness, though I'm unsure how effective it will be for gw2.

**Reply by y0urd0g** [reply]
I appreciate the reply, the problem with color filters is they alter the look of EVERYTHING and end up making the game look wierd, and the only thing i need altered is the AOE circles. but if it cant be done, then it cant be done.
